.github/workflows/docker-image.yml

```diff
@@ -1,8 +1,11 @@
 name: Update GIS docker image tags
```

Comment:
Starting of flow just add what this flow does, 

----

This github workflow will automatically update docker image tags of gis -depl  in the datakaveri/iudx-deployment repository files, whenever docker image is pushed to ghcr.io/datakaveri/gis-depl .
 Based on tag it will update the master/latest branch (if its 5.0.0-alpha-*) or 4.5.0 stable  branch (if its 4,5.0-*)

---